Fixed the locator in Ozon test from 07022020

sshitov opened the pull request:
Hello, I fixed the locator in the Ozon test. The test runs successfully. 
Proof: https://monosnap.com/file/6LSedtzZdHQbJsTD6u0vlqTvikVmRV

sshitov committed:
Fixed the locator in Ozon test from 07022020

KyuzalovaKarina commented on src/test/java/Test_Git.java:
```diff
@@ -26,11 +26,11 @@ public void testSampleCase() throws Exception {
         //кнопка в корзину
-        driver.findElement(By.xpath("//div[@id='__layout']/div/div/div[2]/div[2]/div[2]/div[2]/div/div/div/div/div/div/div[3]/div/button")).click();
-        driver.findElement(By.cssSelector("svg.a8n")).click();
+        driver.findElement(By.xpath("//button[@qa-id='tile-buy-button']")).click(); // Fixed the locator here
```
Данный XPath у всех кнопок в корзину. Если сделаешь findelements то увидишь. Нужен уникальный. Можно рассмотреть вариант привязки к названию книги

KyuzalovaKarina commented on src/test/java/Test_Git.java:
```diff
@@ -26,11 +26,11 @@ public void testSampleCase() throws Exception {
         //кнопка удалить из корзины
-        driver.findElement(By.xpath("//div[@id='__layout']/div/div/div/div/div[3]/div[5]/div/div/div/div[2]/div[3]/div[2]/div[2]/span[2]")).click();
-        driver.findElement(By.xpath("//div[3]/button/div")).click();
+        driver.findElement(By.xpath("//div[@class='column md-8']//span[2]")).click(); // Fixed the locator here
```
Есть подозрение что атрибут class динамический. Вечером проверю

sshitov committed:
Fixed the locators in Ozon test from 09022020
- Clarified the first element in the list
- Fixed CSS locator for the basket
- Fixed XPath locator for the "delete" button in popup
